File: 231217/짝수이면서 합이 5의 배수인 수/an-even-number-with-a-multiple-of-5-in-the-sum.js
function isMagicNumber(n) {
    return n % 2 === 0 && (parseInt(n / 10) + (n % 10)) % 5 === 0;
}

// 변수 선언 및 입력
const fs = require("fs");
let n = Number(fs.readFileSync(0).toString().trim());

if (isMagicNumber(n)) {
    console.log("Yes");
}
else {
    console.log("No");
}

File: 231217/짝수이면서 합이 5의 배수인 수/an-even-number-with-a-multiple-of-5-in-the-sum.py
# 변수 선언 및 입력:
n = int(input())

def is_magic_number(n):
    return n % 2 == 0 and (n // 10 + (n % 10)) % 5 == 0

if is_magic_number(n):
    print("Yes")
else:
    print("No")